frontend/src/v4/routes/jobs/jobs.component.tsx

```diff
@@ -32,7 +32,7 @@ import { NewJobForm } from '../components/newJobForm/newJobForm.component';
 import { UserManagementTab } from '../components/userManagementTab/userManagementTab.component';
 import { LoaderContainer } from '../userManagement/userManagement.styles';
 import { Panel } from '../components/floatingActionPanel/floatingActionPanel.styles';
-import { Container, NewJobBottomButton } from './jobs.styles';
+import { Container } from './jobs.styles';
 
 const JOBS_TABLE_CELLS = [{
 	name: 'Job name',
```

Comment:
At line 165, you can get rid of `openNewJobForm`. That was introduced for the bottom button and only used by that